examples/jupyter-tutorials/API.ipynb

```diff
@@ -11,188 +11,1039 @@
-    "lp = m.add_lookup_parameters((3,5), init=0)                         # Creates 3 vectors of dimension 5 filled with zeros\n",
-    "lp = m.add_lookup_parameters((3,5), init='uniform', scale=a)        # Creates 3 vectors of dimension 5 initialized with U([-a,a])\n",
-    "lp = m.add_lookup_parameters((3,5), init='normal', mean=a, std=b)   # Creates 3 vectors of dimension 5 initialized with N(a, b)\n",
-    "lp = m.add_lookup_parameters((3,5), init='glorot')                  # Creates 3 vectors of dimension 5 with glorot init\n",
-    "lp = m.add_lookup_parameters((3,5), init='he')                      # Creates 3 vectors of dimension 5 with he init\n",
-    "lp = m.add_lookup_parameters(np.zeros((3,5)))                       # Creates 3 vectors of dimension 5 from a numpy array (first dimension is the lookup dimension)\n",
-    "lp = m.add_lookup_parameters((3,5), init=dy.PyInitializer())        # Any parameter initializer\n",
+    "e5.set(10)  # now the e5 expression contains row 10\n",
+    "print('e5 dim after applying set method', e5.dim())\n",
+    "print(e5.value())\n",
     "\n",
-    "## ===== Combine expression into complex expressions.\n",
+    "# We can check if it is actually containing row 10\n",
+    "e10 = lp[10]\n",
+    "print(e5.value() == e10.value())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Similar to Parameters, we have several ways to\n",
+    "# initialize LookupParameters.\n",
+    "scale, mean, stddev = 1, 0, 1\n",
     "\n",
-    "# Math \n",
-    "e = e1 + e2   \n",
-    "e = e1 * e2   # for vectors/matrices: matrix multiplication (like e1.dot(e2) in numpy)\n",
-    "e = e1 - e2    \n",
-    "e = -e1 \n",
+    "# Creates 3x5 matrix filled with 0 (or any other float)\n",
+    "p = m.add_lookup_parameters((3,5), init=0)\n",
+    "# Creates 3x5 matrix initialized with U([-scale, scale])\n",
+    "p = m.add_lookup_parameters((3,5), init='uniform', scale=scale)\n",
+    "# Creates 3x5 matrix initialized with N(mean, stddev)\n",
+    "p = m.add_lookup_parameters((3,5), init='normal', mean=mean, std=stddev)\n",
+    "# Creates 5x5 identity matrix\n",
+    "p = m.add_lookup_parameters((5,5), init='identity')\n",
+    "# Creates 3x5 matrix with glorot init\n",
+    "p = m.add_lookup_parameters((3,5), init='glorot')\n",
+    "p = m.add_parameters((3,5)) # By default, the init = 'glorot'\n",
+    "# Creates 3x5 matrix with he init\n",
+    "p = m.add_lookup_parameters((3,5), init='he')\n",
+    "# Creates 3x5 matrix from a numpy array (size is inferred)\n",
+    "p = m.add_lookup_parameters((3,5), np.ones((3,5)))\n",
+    "# Creates 3x5 matrix from a numpy array (size is inferred)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### More Expression Manipulation\n",
+    "DyNet provides hundreds of operations on Expression. User can manipulate Expressions, build complex Expression easily."
```

Comment:
operations on Expression -> operations that can be applied to Expressions

User -> The user

build complex Expression -> and build complex Expressions